src/sentry/static/sentry/app/components/dropdownAutoComplete.jsx

```diff
@@ -59,7 +59,10 @@ class DropdownAutoComplete extends React.Component {
 
   filterItems = (items, inputValue) =>
     items.filter(item => {
-      return item.value.toLowerCase().indexOf(inputValue.toLowerCase()) > -1;
+      return (
+        item.value.toLowerCase().indexOf(inputValue.toLowerCase()) > -1 ||
+        item.label.toLowerCase().indexOf(inputValue.toLowerCase()) > -1
```

Comment:
https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/assigneeSelector.jsx#L44

other places we join the strings then do the conversions (nit)